core/domain/config_services_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing Services for configuration properties."""
+
+import collections
+
+from core.domain import config_domain
+from core.domain import config_services
+from core.tests import test_utils
+
+
+class ConfigServicesDomainUnitTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ConfigServicesDomainUnitTests, self).setUp()
+        ConfigObj = collections.namedtuple(
+            'ConfigObj', 'name new_config_value')
+        self.committer_id = u'configServicesCommitterID'
+        self.config_obj = ConfigObj(
+            name='propertyName', new_config_value='propertyValue')
+        self.invalid_property_name = u'_no_such_property_name'
+        self.property_dummy_value = u'TEST_VALUE'
+        self.schema_unicode = 'unicode'
+        self.cfg_schemas = config_domain.Registry.get_config_property_schemas()
```

Comment:
We try not to use abbreviations in names. FYI, in case line length is an issue, you could also do:

```
    self.cfg_schemas = (
        config_domain.Registry.get_config_property_schemas())
```

Also, this isn't really needed throughout the tests, you only need the first property name. So don't attach this to self, but you can set up self.default_property_value or something.